Sources/MediaViewController.swift

```diff
@@ -23,6 +23,8 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
     private var mediaDatasourceAndDelegate: MediaDataSourceAndDelegate?
     private var searchController: UISearchController?
     private let searchDataSource = VLCLibrarySearchDisplayDataSource()
+    private var rendererButton: UIButton
```

Comment:
you don't need this

Reply:
I'm not sure, I opted for a implementation of the renderer button by the VC that needs the renderer mostly because I want the possibility to the VC to customize their button if needed.
I'm not really happy by it because I  think the buttons should be contained in the manager but I didn't have a better idea. 😞 
For example: `MediaVC` and `VLCMovieVC` buttons have a different behaviour.

Reply:
when do they need to customize the button ? if they don't need get rid of this and for different behaviors you have addaction target and remove action target